docs/sensors/cloudino.md

```diff
@@ -1,9 +1,392 @@
+  *Fig. 2. Cloudino.io Portal*
+
+Firstly you need to create an account and login to the platform where you will
+find a configuration section to manage Cloudino devices, clouding rules and
+[FIWARE Orion Context Broker (OCB)](https://fiware-orion.readthedocs.io)
+connections as well as a main Getting Started guide.
+
+  ![Login page](./images//cdn_03.png)
+  *Fig. 3. Login page*
+
+  ![Cloudino.io User's Main page](./images//cdn_04.png)
+  *Fig. 4. Cloudino.io User's Main page*
+
+To connect your air quality sensor unit to the Cloudino platform, the Cloudino
+Wifi Connector needs to be set up in the portal. **CWC is the hardware with Cloudino
+tecnology that allows you to connect any device to IoT**.
+
+  ![Cloudino Wifi Connector](./images//cdn_05.png)
+  *Fig. 5. Cloudino Wifi Connector*
+
+From the left side panel, select the Devices Menu and click the "Add Device" option.
+Then introduce a name, description and hardware type configuration; In this
+case, an Arduino UNO will be used for developing our air quality sensor unit as
+seen on Fig. 1. *You must select the proper hardware type (Arduino) to use from
+the list or use Cloudino as standalone device*.
+Click on the "Submit" button and our device will be created.
+You can register as many CWCs devices as you need.
+
+  ![CWC device registed](./images//cdn_06.png)
+  *Fig. 6. CWC device registered*
+
+After creation, several sections will be shown in order to configure, control
+and program our device as seen on Fig. 8.
+
+  ![CWC General Information](./images//cdn_07.png)
+  *Fig. 7. CWC General Information*
+
+The General View shows the device's id, authentication token (Auth Token), name,
+description, type and security level (public or private). This information will
+be used later to associate our CWC to the device created in the portal.
+
+### Connect a Cloudino Wifi Connector to Cloudino.io Portal
+
+Once you have created a device in the portal, it is necessary to associate the
+CWC to it by attaching the Auth Token generated in the previous step.
+In this way, our air quality unit will be connected to the Cloudino platform.
+
+To perform this action, connect the CWC to a power source (5 volts). It will
+create a WiFi Network on which we need to connect to access the CWC configuration
+panel. **The WiFi network's SSID is the serial number of the CWC device and is
+created without any password**.
+
+  ![WiFi network generated by any CWC](./images//cdn_08.png)
+  *Fig. 8. WiFi network generated by any CWC*
+
+For instance, Fig. 8 highligths Cloudino_FAAE37 network that is used to
+configure the particular CWC (Cloudino_FAAE37 device).
+
+As soon as the WiFi connection is made, a configuration panel shows up. In case
+it doesn't happen, you can access to the panel using a web browser with the
+**192.168.4.1** IP address. Fig. 9 shows the main configuration Web page for a CWC.
+
+  ![CWC Configuration Panel on 192.168.4.1](./images//cdn_09.png)

```

Comment:
is this a fixed IP on cloudino right? so it's always the same ip?

Reply:
Yes, this is a fixed local IP from Cloudino and is always the same. It usually access directly once Cloudino is connected